SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.Windows.Shared/Channel/SessionTracker.cs

```diff
@@ -89,15 +93,15 @@ private void HandleEnqueuingLog(object sender, EnqueuingLogEventArgs e)
                     return;
                 }
                 SendStartSessionIfNeeded();
-                e.Log.Sid = Sid == Guid.Empty ? null : new Guid?(Sid);
+                e.Log.Sid = Sid;
                 _lastQueuedLogTime = TimeHelper.CurrentTimeInMilliseconds();
             }
         }
 
         private void SendStartSessionIfNeeded()
         {
             var now = TimeHelper.CurrentTimeInMilliseconds();
-            if (Sid != Guid.Empty && !HasSessionTimedOut(now))
+            if (_lastQueuedLogTime > 0 && !HasSessionTimedOut(now))
```

Comment:
Since Sid is static, to test if we ever send a log in this instance we can use this field instead.